app/templates/main/uncaughtValidation.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
```

Comment:
Could we update the (outdated) contents of `app/templates/main/500.html` and use that instead grather than creating another view for this case?